### ISSUE TITLE
Footer

### ISSUE THREAD

**Issue by IngridGdesigns** [IngridGdesigns/Urban_Garden#10]
Footer has weird behavior in my pages, sometimes it does not even shows up

**Comment by jwulia** [comment]
Do you want the footer to appear always?

You have a route to the footer, but I don't think that link is being used- https://github.com/IngridGdesigns/Urban_Garden/blob/7b66f9aeef6c76644795511dedc4af5eec66aad0/client/src/components/Routes.js#L59

The footer is loaded in the App component but not the other routes- https://github.com/IngridGdesigns/Urban_Garden/blob/7b66f9aeef6c76644795511dedc4af5eec66aad0/client/src/components/Routes.js#L43

**Comment by IngridGdesigns** [comment]
Yes, I added that component to my pages but I think I may have a bug. Still working though it. Tomorrow we are collaborating with others in the cohort on our projects, so I was told to add issues so peoplel can work on them.
